builds/jazz-build-module/aws-lambda-events-module.groovy

```diff
@@ -8,12 +8,73 @@ import groovy.transform.Field
 */
 
 @Field def config_loader
-
+@Field def queue_visibility_timeout = 165
 
 def initialize(configLoader){
   config_loader = configLoader
 }
 
+def checkSQSAndAddLambdaTrigger(queueName, lambdaARN) {
+  try {
+    sh "aws sqs get-queue-url --queue-name $queueName --profile cloud-api --output json"
+    echo "Queue exists and have access"
+    addLambdaTriggerToSqsQueue(true, queueName, lambdaARN)
```

Comment:
This is actually a tricky flow logic and can go wrong.  Since the flow logic is relied on an exception and it can happen for various reasons.  Best way would be,  parse the response and validate the proceed next.  Also for catch logic,  see if it failed for correct reason, validate the error message (since flow here proceeds to create an SQS queue